pvdeg/standards.py

```diff
@@ -14,80 +14,183 @@
+    wind_factor=0.33,):
+
+    """
+    Calculate and set up data necessary to calculate the effective standoff distance for rooftop mounded PV system 
+    according to IEC TS 63126. The the temperature is modeled for T_0 and T_inf and the corresponding test
+    module temperature must be provided in the weather data.
+
+    Parameters
+    ----------
+    weather_df : pd.DataFrame
+        Weather data for a single location.
+    meta : pd.DataFrame
+        Meta data for a single location.
+
+    sky_model : str, optional
+        Options: 'isotropic', 'klucher', 'haydavies', 'reindl', 'king', 'perez'.
+    temp_model : str, optional
+        Options: 'sapm'.  'pvsyst' and 'faiman' will be added later.
+        Performs the calculation for the cell temperature.
+    conf_0 : str, optional        Model for the high temperature module on the exponential decay curve.
+        Default: 'insulated_back_glass_polymer'
+    conf_inf : str, optional
+        Model for the lowest temperature module on the exponential decay curve.
+        Default: 'open_rack_glass_polymer'
+    wind_factor : float, optional
+        Wind speed correction exponent to account for different wind speed measurement heights
+        between weather database (e.g. NSRDB) and the tempeature model (e.g. SAPM)
+        The NSRDB provides calculations at 2 m (i.e module height) but SAPM uses a 10 m height.
+        It is recommended that a power-law relationship between height and wind speed of 0.33 
+        be used*. This results in a wind speed that is 1.7 times higher. It is acknowledged that 
+        this can vary significantly. 
+    
+    R. Rabbani, M. Zeeshan, "Exploring the suitability of MERRA-2 reanalysis data for wind energy 
+        estimation, analysis of wind characteristics and energy potential assessment for selected
+        sites in Pakistan", Renewable Energy 154 (2020) 1240-1251.
+
+    Meta data
+    ------------------
+    tilt : float, 
+        Tilt angle of PV system relative to horizontal. [°] Required
+    azimuth : float,
+        Azimuth angle of PV system relative to north. [°] Required
+    Wind_Height_m : float,
+        if wind_factor is "None", it will run a calculation based on the height and a 
+        power factor of 0.33. If neither are supplied, the wind speed will not be adjusted.
+
+    Returns
+    -------
+    T_0 : float
+        An array of temperature values for a module with an insulated back or an 
+        alternatively desired small or zero standoff, [°C]. Used as the basis for the 
+        maximum achievable temperature.
+    T_inf : float
+        An array of temperature values for a module that is rack mounted, [°C].
+    T_measured : float
+        An array of values for the test module in the system, [°C] interest.
+    poa : float
+        An array of values for the plane of array irradiance, [W/m²]
+
+    """
+
+    parameters = ["temp_air", "wind_speed", "dhi", "ghi", "dni", "Module_Temperature"]
```

Comment:
the main purpose of this list is to only load the necessary parameters for the analysis to reduce the time it takes to load the data for geospatial analysis.

Reply:
This method is doing a special calculation. To calculated x_eff you need to measure the actual module temperature along with the irradiance and wind speed to be able to calculate the theoretical insulated back temperature and open rack temperature. So for this function we are not using satellite data, but are using data from an actual test location not from the NSRDB which does not have a module temperature.

Is this clear now?

Reply:
Okay this makes sense, but why would you add the measured module temperature to the weather data frame? Shouldn't this just be it's own input? That way, users can use whatever meteorological data they have and include the measured module temperature. 

By adding module temperature to the weather dataframe this functions breaks with the conventions used in all the other functions. So, I'm trying to understand what benefit this has to combine the measured temperature with the weather data. If you only want to provide one dataframe, we should rename it to `measured_df` or something like this. So, it's immediately clear that this is different to what is used in the other functions. 

Also lines 90 to 96 won't work with the parameter list as it's written now.

Reply:
I just wanted to use the same format so I put it in a weather data frame. I thought this would facilitate other uses that might be envisioned in the future. Just making all the variables to be formatted in the same way. The module data needs to be matched up with the meteorological data. Also, I found that the read file function was fully functional enough to pull in another column of data easily. This is a good feature for flexibility just in case someone else wants to add another type of data such as voltage, shading, rain, snow, or anything else in a custom supplied data set. We also need to consider that any of these functions should work with custom ground based data sets.

Reading the next paragraph, are you just saying I should change the name? I'm fine with that. Maybe that is better just from an ability to understand the code. But technically both are measured, one is based on custom ground based data. Let me know your thoughts and I can fix this either was very quickly. Maybe there is a third option for a variable name?

Reply:
okay, I'll suggest renaming it so that there's no confusion with the weather dataframes.